core/services/workflows/engine_test.go

```diff
@@ -24,15 +24,15 @@ import (
 
 const hardcodedWorkflow = `
 triggers:
-  - id: "mercury-trigger"
+  - id: "mercury-trigger@1.0.0"
```

Comment:
All these should be merged now, please rebase.

Reply:
Handled.